app/javascript/components/my-studies/StudyUsageInfo.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from 'react'
+import _uniqueId from 'lodash/uniqueId'
+import Plotly from 'plotly.js-dist'
+
+import { fetchStudyUsage } from '~/lib/scp-api'
+import LoadingSpinner from '~/lib/LoadingSpinner'
+import { supportEmailLink } from '~/lib/error-utils'
+import InfoPopup from '~/lib/InfoPopup'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronUp, faChevronDown } from '@fortawesome/free-solid-svg-icons'
+
+/** display mixpanel stats for a given study */
+export default function StudyUsageInfo({ study }) {
+  const [usageInfo, setUsageInfo] = useState(null)
+  const [isLoading, setIsLoading] = useState(true)
+  const [showTable, setShowTable] = useState(false)
+  const [viewsGraphElementId] = useState(_uniqueId('study-usage-graph-'))
+  const [downloadsGraphElementId] = useState(_uniqueId('study-usage-graph-'))
+  const studyAccession = study.accession
+
+  /** get the usage information via API call */
+  useEffect(() => {
+    fetchStudyUsage(study._id.$oid).then(response => {
```

Comment:
While I agree with Eric's point, there is somewhat of a gotcha here - because accessions are not _globally_ unique (they're only unique to a given instance), you can get pollution of stats across dev/staging instances.  I confirmed this by booting my local instance and looking at the stats for a clone of SCP1 from production (sNuc-seq study, Naomi's data).  In my local instance, this is SCP13.  When I looked at the stats, it registered views & downloads for April, which obviously haven't happened as I only just booted my server again today.

That being said, I think it's only an issue in local dev & staging, and should not affect production as this uses a different Mixpanel instance.  But we should at least be aware of the issue.

Reply:
That's a keen observation!  We can distinguish staging and development by adding an `env` property with value `staging` or `development` to our logs, but your points stands at least _within_ different development environments.

We don't log the study's MongoDB object ID to Mixpanel for these events, IIRC, so using accession in this SCP API wouldn't affect that matter.  Less persuasively, but maybe also worth noting: which study property is queried in Mixpanel also seems somewhat orthogonal to the type of identifier used in the SCP API.

If we deem intra-development environment Mixpanel log pollution worth fixing, I'd be inclined to do so by adding a new property, perhaps something like `instance`, to all logged event.  This would help distinguish all events across instances within an environment, not just those for a specific study.

Reply:
Inter-dev-environment pollution can also be addressed by breaking results out by Mixpanel distinct user id, leveraging the fact that we don't use each other's instances